CODING_STANDARD

```diff
@@ -171,6 +171,20 @@ Output:
   (status(), error(), debug(), etc)
 - Use '\n' instead of std::endl
 
+Unit tests:
+ - Unit tests are written using Catch: https://github.com/philsquared/Catch/
+ - For large classes:
+    - Each file should contain tests for one function in one class
```

Comment:
I've just edited my suggestion again, which shows that it's hard to find clear wording for this!